examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -24,10 +24,20 @@ export default mudConfig({
     },
   },
   modules: [
+    {
+      name: "KeysInTableModule",
+      root: true,
+      args: [resolveTableId("CounterTable")],
+    },
```

Comment:
if we have capacity before wednesday we could add a super simple plugin that converts the config by adding this module for all tables in the config